content/pt-br/serverless.md

```diff
@@ -0,0 +1,17 @@
+---
+Title: Serverless
+Status: Completed
+Category: Technology
```

Comment:
Other `pt-br` terms are using the category "tecnologia".

- https://github.com/cncf/glossary/blob/main/content/pt-br/container.md?plain=1#L4
- https://github.com/cncf/glossary/blob/main/content/pt-br/containerization.md?plain=1#L4
- ...

```suggestion
Category: tecnologia
```